### PR TITLE
Added new policy for Port 22 check

### PR DESCRIPTION
**Pull request by mavericknavs**

### CODE REVIEW

**Comment by mavericknavs** [comment]
@jerbia Can You please have a look?

File: docker/policies/port22.rego [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    input[i].Cmd == "EXPOSE"
+    val := input[i].Value
+    contains(val[_], denylist[_])
+}
+
+deny[res] {
+  failPortCheck
+  msg := sprintf("Specify Port to SSH into the container")
+  res := {
+  	"msg": msg,
```
Comment (knqyf263) [inline docker/policies/port22.rego:28]: Please apply `opa fmt`
https://www.mankier.com/1/opa-fmt

File: docker/policies/port22.rego [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "severity": "Medium",
+    "type": "Dockerfile Security Check",
+    "description": "Exposing Port 22 allows users to SSH inside the container.",
+    "recommended_actions": "Remove port 22 from the dockerfile",
+}
+
+denyList =[22] 
+
+# failPortCheck is true if the image contains an expose statement for value 22
+# tag is latest.
```
Comment (knqyf263) [inline docker/policies/port22.rego:16]: cruft?

Comment (jerbia) [reply docker/policies/port22.rego:16]: What do you mean by "tag is latest"? Seems like a copy & paste issue.

**Comment by mavericknavs** [comment]
As per the comments - I've run opa fmt as well as opa test as well to test the changes.

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**